cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Startup.cs

```diff
@@ -38,6 +38,8 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
+            Console.Write("### App Version:{0} ###", Assembly.GetExecutingAssembly().GetName().Version);
+
```

Comment:
Note that this is redundant; there is an application version info service that provides the full details on the assembly.